src/app/shared/components/template/components/index.ts

```diff
@@ -197,3 +195,8 @@ export const TEMPLATE_COMPONENT_MAPPING: Record<
   video: TmplVideoComponent,
   workshops_accordion: WorkshopsComponent,
 };
+
+export const TEMPLATE_COMPONENT_MAPPING = {
+  ...CORE_COMPONENT_MAPPING,
+  ...DEMO_COMPONENT_MAPPING,
```

Comment:
@jfmcquade 
nit(non-blocking): realised after merge, but might be good to also include the plh components mapping here as well for any new components added to the placeholder module